tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ClassEmitter.scala

```diff
@@ -133,8 +133,43 @@ private[emitter] final class ClassEmitter(jsGen: JSGen) {
       yield js.ClassDef(Some(classIdent), parentVar, members)(tree.pos)
   }
 
+  /** Extracts the inlined init constructor, if there is one. */
+  def extractInlinedInit(tree: LinkedClass)(
```

Comment:
Consider "inlineable" (here and in many places). "inlined" sounds like it has already happened. If you are concerned about it sounding like a choice, at least improve the doc.